samples/ChatSample/ChatSample.csproj

```diff
@@ -15,6 +15,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.AspNetCore.SignalR\Microsoft.AspNetCore.SignalR.csproj" />
+    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.SignalR.Client.TS\Microsoft.AspNetCore.SignalR.Client.TS.csproj" />
```

Comment:
Try adding `ReferenceOutputAssembly="false"` to all ProjectReference to the TS project. This will suppress NuGet attempting to use the project reference as a compile reference.